src/sage/graphs/dot2tex_utils.py

```diff
@@ -74,7 +74,7 @@ def quoted_latex(x):
 
     EXAMPLES::
 
-        sage: sage.graphs.dot2tex_utils.quoted_latex(matrix([[1,1],[0,1],[0,0]]))
+        sage: sage.graphs.dot2tex_utils.quoted_latex(matrix([[1,1],[0,1],[0,0]]))     # optional - sage.modules
```

Comment:
in this file, the alignement of `# optional - sage.modules` is different than in other files